feat: Editorial review for Shared Storage API docs (#30427)
mdn/content: files/en-us/web/api/sharedstorageworkletglobalscope/sharedstorage/index.md

---
title: "SharedStorageWorkletGlobalScope: sharedStorage property"
short-title: sharedStorage
slug: Web/API/SharedStorageWorkletGlobalScope/sharedStorage
page-type: web-api-instance-property
status:
  - experimental
browser-compat: api.SharedStorageWorkletGlobalScope.sharedStorage
---

{{APIRef("Shared Storage API")}}{{SeeCompatTable}}

> The **`context`** read-only property of the
> {{domxref("SharedStorageWorkletGlobalScope")}} interface contains a {{domxref("WorkletSharedStorage")}} object instance representing the shared storage for a particular origin, as it is exposed to a worklet context.

Reviewer: Would it be correct to say:
```suggestion
The **`context`** read-only property of the
{{domxref("SharedStorageWorkletGlobalScope")}} interface contains a {{domxref("WorkletSharedStorage")}} object instance, which represents the shared storage for a particular origin as exposed to a worklet context.
```
